src/pages/Manager.vue

```diff
@@ -644,8 +644,20 @@ import CategoryFilterModal from '../components/modals/CategoryFilterModal.vue';
             }
         }
 
-        beforeCreate() {
+        async beforeCreate() {
             this.activeGame = GameManager.activeGame;
+
+            // Used by SearchAndSort, but need to be called here to
+            // ensure the settings are loaded before LocalModList
+            // accesses visibleModList from Vuex store.
+            await this.$store.dispatch('profile/loadOrderingSettings');
+
+            // Reset the mod list to prevent the previous profile's list
+            // flashing on the screen while a new profile's list is loaded.
+            await this.$store.dispatch('profile/updateModList', []);
+
+            // Used by OnlineModView, called here for consistency.
+            this.$store.commit('modFilters/reset');
```

Comment:
Should this `await`?

Reply:
`.dispatch` (actions) need to await, `commit` (mutations) are synchronous methods.